### PR TITLE
fix: fixed lower case for "IN" operator in queryFilters [DHIS2-11770]

### PR DESCRIPTION
**Pull request by gnespolino**
This should go into 2.37.

Adding @Philip-Larsen-Donnelly for approval.

### CODE REVIEW

**Review by maikelarabori — APPROVED** [review]
Looks good @gnespolino :+1: 

Thanks

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/InQueryFilter.java [hunk 1]
```diff
@@ -63,13 +65,16 @@ public InQueryFilter( String field, QueryFilter queryFilter )
      * @param encodedFilter actual "in" parameters
      * @return a SQL condition representing this InQueryFilter
      */
-    public String getSqlFilter( String encodedFilter )
+    public String getSqlFilter( String encodedFilter, boolean isText )
```
Comment (larshelge) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/InQueryFilter.java:68]: Please add a description to explain the initial problem and why this fixes it :-)

**Review by stian-sandvold — APPROVED** [review]

**Comment by larshelge** [comment]
Looks good @gnespolino ! Thanks for the explanation/description.

**Comment by larshelge** [comment]
I think we could have added a few unit tests here, especially `InQueryFilter` should be feasible to unit test.

**Pull request merged**